src/commands/app/db/collection/updateOne.js

```diff
@@ -52,30 +36,26 @@ export class UpdateOne extends DBBaseCommand {
       }
 
       // Update the document
-      const result = await coll.updateOne(filterObj, updateObj, options)
+      const result = await coll.updateOne(filter, update, options)
 
       this.debugLogger?.info?.('Document updated successfully:', result)
 
       const response = {
         collection,
         filter: filterObj,
         update: updateObj,
```

Comment:
`filterObj` and `updateObj` don't exist anymore.

```suggestion
        filter,
        update,
```